docs/manuals/running_services/index.md

```diff
@@ -43,10 +43,10 @@ if __name__ == "__main__":
     rest.run()
 
     # Approach 2
-    tp.run([core, rest], title="Taipy Demo")
+    tp.run(core, rest, title="Taipy Demo")
```

Comment:
I haven't seen this syntax yet. But it it works, it works.
In the situation where you have more than one service, the array syntax is not such a bit deal to me.

Reply:
I have tested and checked the code again, it works for both with array or no array so we should be good :D